### DISCUSSION TITLE
404 loading resource generated by bundle extension

### DISCUSSION THREAD

**Discussion by spotthehall** [datenstrom/community#563] [Report a bug] [answered]
With Datenstrom Yellow 0.8.16, PHP 7.4.9, Apache 2.4.46, I am getting a 404 error when trying to load the extension bundle: /media/extensions/bundle-****.min.css

Two site effected, so far, They work Ok on a fresh upload, but in use fail.

**Comment by spotthehall** [comment]
A way I have got around just replacing the extensions folder every time if fails, is (I think) to build the site locally, upload everything, change permissions on the extensions folder to 777 let it write the bundle, then change the permissions back to 775.

**Comment by markseuffert** [comment, marked as answer]
Hello, I updated the way how bundles are generated and cleaned up. I suspect that the cleaning up part was the problem on your web server, I think it deleted too much. Now the [bundle extension](https://github.com/annaesvensson/yellow-bundle) should work without 404 error.

Can you update your website and try it out?

**Comment by spotthehall** [comment]
Thanks markseu, I installed the update yesterday on both sites and everything seems to be working just fine now.
Cheers.

**Comment by markseuffert** [comment]
Thanks for testing and giving feedback. 😀
